apps/test/unit/lib/kits/maker/boards/makeStubBoard.js

```diff
@@ -17,11 +17,6 @@ export class MicrobitStubBoard {
     this.updateListeners = [];
     this.analogChannel = new Array(16).fill(0);
     this.myPort = null;
-    this.boardClient_ = null;
```

Comment:
This was added by [this PR](https://github.com/code-dot-org/code-dot-org/pull/50258). At the time, I didn't understand that `MicrobitStubBoard` stubs `MBFirmataWrapper`, not `MicroBitBoard`. So removing this value.
For context, in `MicroBitBoard`, `this.boardClient_` is a `MBFirmataWrapper`.